extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -149,7 +149,7 @@ function PanelStudyBrowserTracking({
       const imageId = imageIds[Math.floor(imageIds.length / 2)];
 
       // TODO: Is it okay that imageIds are not returned here for SR displaysets?
-      if (imageId) {
+      if (imageId && !displaySet?.unsupported) {
```

Comment:
just remember that you should handle PanelStudyBrowser too (not tracking the vanilla one)

Reply:
Its there also https://github.com/rodrigobasilio2022/Viewers/blob/f79d56f66c37e3ab0b90fd62d4014c9633fb35d3/extensions/default/src/Panels/PanelStudyBrowser.tsx#L126